File: MACHINE RUN 06-11-2020/Simple interest/simple interest.c
#include<iostream>
using namespace std;
class Simple_interest
{
    int P,r,t;
public:
    void display()
    {

        cout<<"Enter the Principal amount";
        cin>>P;
        cout<<"Enter the Interest rate";
        cin>>r;
        cout<<"Enter the Time";
        cin>>t;
        cout<<"Final amount is;"<<P+P*r*t;



    }};
    int main()
    {
        Simple_interest s1;
        s1.display();
    }

File: ASSIGNMENT 1(java)/Simpleinterest.java
import java.util.Scanner; 
public class SimpleInterest
{
 public static void main(String[] args) 
{
        
        int P,r,t,f;
        
        System.out.println("Enter The Principal Amount: ");
        Scanner sc = new Scanner(System.in);
        P = sc.nextInt();
        System.out.println("Enter The Interest Rate: ");
        Scanner sb = new Scanner(System.in);
        r = sb.nextInt();
        System.out.println("Enter The Timeperiod: ");
        Scanner sa = new Scanner(System.in);
        t = sa.nextInt();
        f=P+(P*r*t);
        System.out.println("The Final Amount  is: "+f);
      
    
    }
}
